### ISSUE TITLE
Plugin installer hangs at loading

### ISSUE THREAD

**Issue by ascclemens** [goatcorp/Dalamud#950]
A dev plugin failed to reload, so I went to disable it, but it looks like the installer is just bricked after. It went up to `PluginMasters reloaded; now refiltering...` in the log. 6.4.0.42 (`a585c3f`).

![image](https://user-images.githubusercontent.com/955250/182508948-6f292b93-b859-49dd-a1ec-4cfa6f38db66.png)

**Comment by goaaats** [comment]
No exceptions in the log?

**Comment by ascclemens** [comment]
Nope, aside from the plugin failing to (re)load. It works fine if it's just a normal failed load; no idea if it's related to the reload or not.

**Comment by Aireil** [comment]
Can't seem to reproduce this, just to be sure, are these steps correct?
1. Dev plugin was loaded properly.
2. Dev plugin got reloaded and errored during loading.
3. You opened the installer for the first time of that Dalamud instance.

**Comment by ascclemens** [comment]
Yeah, that's what happened. Not sure if that was the first open of the installer. I think that's doubtful. Maybe it was just a freak accident.
